src/components/explore/TranscriptContentPage.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, useEffect, useRef, useState } from "react";
+import BreadCrumbs from "../common/BreadCrumbs";
+import GroupedTranscriptContent from "./GroupedTranscriptContent";
+import AlphabetGrouping from "./AlphabetGrouping";
+import {
+  createSlug,
+  DepreciatedCategories,
+  groupDataByAlphabet,
+  sortKeysAlphabetically,
+} from "@/utils";
+import MobileAlphabetGrouping from "./MobileAlphabetGrouping";
+import ContentGrouping from "./ContentGrouping";
+
+interface ITranscriptContentPage {
+  header: string;
+  description?: string;
+  mobileDescription?: string;
+  data: any /* A Json that changes could be topics, speakers, categories  */;
```

Comment:
I think we can define the type of our JSON in a latter PR

Reply:
Yh, should i do it in this PR?

Reply:
> Yh, should i do it in this PR?

another PR will handle it, sources is using the component and I don't want setting types here to disrupt that